packages/management/framework/package.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: framework.tanzu.vmware.com
+  namespace: framework
+spec:
+  refName: framework.tanzu.vmware.com
+  version: VERSION
```

Comment:
I see other packages have `latest` here. Should we use the same for consistency?